### PR TITLE
[DOCSS-1047] Cleanup jekyll unused parts

### PR DESCRIPTION
**Pull request by rsylvian**
Ticket: [DOCSS-1047](https://circleci.atlassian.net/browse/DOCSS-1047)

# Description
- Removed/Archived all unused `_layouts`
- Removed/Archived all unused `_includes`
- Uninstall `jekyll-algolia`, `jekyll-include-cache`, `jekyll-target-blank`, `jekyll-toc`, `jekyll-last-modified-at`, `jekyll-timeago` & `jekyll-redirect-from` gems as these features are now handled by `docs-platform`
- Uninstall various npm packages that are not needed anymore
- Slightly edit content based on errors surfaced by HTMLProofer ( mainly `redirect_from`) - Thanks @rosieyohannan for the great pairing session on this 👍 

# Reasons
Multiple Jekyll `_layouts`, `_includes`, yarn & gem packages are not needed anymore. The goal of this PR is to clean this up to have a cleaner open-source repo that is easier to contribute to.

# Content Checklist
N/A

[DOCSS-1047]: https://circleci.atlassian.net/browse/DOCSS-1047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: jekyll/_cci2/api-developers-guide.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ title: "CircleCI API Developer's Guide"
...
 description: "API guide for internal and external CircleCI developers"
 categories: [getting-started]
-contentTags: 
+contentTags:
   platform:
   - Cloud
 ---
 
-This API Developer's Guide was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows. The API v2 Specification itself may be viewed in the [Reference documentation]({{site.baseurl}}/api/v2).
+This API Developer's Guide was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows. The API v2 Specification itself may be viewed in the [Reference documentation](https://circleci.com/docs/api/v2/).
```
Comment (rsylvian) [inline jekyll/_cci2/api-developers-guide.md:12]: using full URL so it is rendered as an external link and HTMLProofer can stop complaining about it

File: jekyll/_cci2/projects.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ contentTags:
 ---
 
 
-A CircleCI project shares the name of the associated code repository in your [version control system]({{ site.baseurl }}/gh-bb-integration/) (VCS). Select **Projects** in the CircleCI web app sidebar to enter the projects dashboard. From here you can set up and follow the projects you have access to.
+A CircleCI project shares the name of the associated code repository in your [version control system]({{ site.baseurl }}/github-integration/) (VCS). Select **Projects** in the CircleCI web app sidebar to enter the projects dashboard. From here you can set up and follow the projects you have access to.
```
Comment (rsylvian) [inline jekyll/_cci2/projects.md:14]: @rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

File: jekyll/_cci2/projects.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ For step-by-step guidance, see [Creating a Project in CircleCI]({{site.baseurl}}
 
 Following a project enables a user to subscribe to [email notifications](/docs/notifications/) for the project build status and adds the project to their CircleCI dashboard.
 
-The *Project Administrator* is the user who adds a GitHub or Bitbucket repository to CircleCI as a project. A *User* is an individual user within an org. A CircleCI user is anyone who can log in to the CircleCI platform with a username and password. Users must be added to a [GitHub or Bitbucket org]({{site.baseurl}}/gh-bb-integration/) to view or follow associated CircleCI projects. Users may not view project data that is stored in environment variables.
+The *Project Administrator* is the user who adds a GitHub or Bitbucket repository to CircleCI as a project. A *User* is an individual user within an org. A CircleCI user is anyone who can log in to the CircleCI platform with a username and password. Users must be added to a [GitHub or Bitbucket org]({{site.baseurl}}/github-integration/) to view or follow associated CircleCI projects. Users may not view project data that is stored in environment variables.
```
Comment (rsylvian) [inline jekyll/_cci2/projects.md:29]: @rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

File: jekyll/_cci2/security-recommendations.md [hunk 4]
```diff
@@ -107,4 +107,4 @@ EOF
 {: #see-also }
 {:.no_toc}
 
-[GitHub and Bitbucket Integration]({{ site.baseurl }}/gh-bb-integration/)
+[GitHub and Bitbucket Integration]({{ site.baseurl }}/github-integration/)
```
Comment (rsylvian) [inline jekyll/_cci2/security-recommendations.md:110]: @rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

File: jekyll/_cci2_ja/projects.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ contentTags:
 ---
 
 
-CircleCI のプロジェクトは、お客様の[バージョンコントロールシステム]({{ site.baseurl }}/ja/gh-bb-integration/) (VCS) 内の、関連するコードリポジトリと同じ名前を持ちます。 CircleCI アプリのサイドバーから **Projects** を選択し、プロジェクトダッシュボードに入力します。 ここからアクセス可能なプロジェクトのセットアップやフォローが可能です。
+CircleCI のプロジェクトは、お客様の[バージョンコントロールシステム]({{ site.baseurl }}/ja/github-integration/) (VCS) 内の、関連するコードリポジトリと同じ名前を持ちます。 CircleCI アプリのサイドバーから **Projects** を選択し、プロジェクトダッシュボードに入力します。 ここからアクセス可能なプロジェクトのセットアップやフォローが可能です。
```
Comment (rsylvian) [inline jekyll/_cci2_ja/projects.md:14]: @rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

File: jekyll/_cci2_ja/security-recommendations.md [hunk 6]
```diff
@@ -108,4 +108,4 @@ EOF
 {: #see-also }
 {:.no_toc}
 
-[GitHub と Bitbucket のインテグレーション]({{ site.baseurl }}/ja/gh-bb-integration/)
+[GitHub と Bitbucket のインテグレーション]({{ site.baseurl }}/ja/github-integration/)
```
Comment (rsylvian) [inline jekyll/_cci2_ja/security-recommendations.md:111]: @rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

File: jekyll/_cci2_ja/settings.md [hunk 7]
```diff
@@ -10,7 +10,7 @@ order: 2
 
 CircleCIドキュメントのこの設定セクションに含まれるガイドでは、CircleCI アプリケーションのプロジェクト設定と組織設定のページからアクセスできる機能とタスクについて説明します。 以下のトピックが含まれています。
 
-* [VCS integration]({{ site.baseurl }}/ja/gh-bb-integration/) (GitHub/Bitbucket): アクセス許可とキーに関する情報
+* [VCS integration]({{ site.baseurl }}/ja/github-integration/) (GitHub/Bitbucket): アクセス許可とキーに関する情報
```
Comment (rsylvian) [inline jekyll/_cci2_ja/settings.md:13]: @rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

File: jekyll/_cci2_ja/test-splitting-tutorial.adoc [hunk 8]
```diff
@@ -39,7 +39,7 @@ CircleCI のテスト分割は、Jest、pytest、Maven、 Gradle などの多く
 このチュートリアルに沿って実行するには、以下を準備する必要があります。
 
 * CircleCI のアカウント: アカウントをお持ちでない場合は、 <<first-steps#,無料で登録していただけます。>>
-* CircleCI アカウントに関連付けられたバージョン管理システム (VCS) プロバイダー (GitHub や Bitbucket) : まだお済みでない場合は、<<gh-bb-integration#,GitHub と Bitbucket の連携>> のページの手順に従って VCS プロバイダーの関連付けをしてください。
+* CircleCI アカウントに関連付けられたバージョン管理システム (VCS) プロバイダー (GitHub や Bitbucket) : まだお済みでない場合は、<<github-integration#,GitHub と Bitbucket の連携>> のページの手順に従って VCS プロバイダーの関連付けをしてください。
```
Comment (rsylvian) [inline jekyll/_cci2_ja/test-splitting-tutorial.adoc:42]: @rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks

File: jekyll/_cci2_ja/orbs-faq.md [hunk 9]
```diff
@@ -61,7 +61,7 @@ contentTags:
 
 * **回答:** Orb には[セマンティックバージョニング]()が使用されています。つまり、_メジャー_バージョンを設定すると (例:`3` )、_マイナー_と_パッチ_のすべてのアップデートを受け取ります。静的にバージョンを設定すると(例:`3.0.0` )、アップデートは適用されません。これが最も決定論的で推奨される方法です。
 
-_**注: **<非推奨> `@volatile` を使って最新バージョンの Orb を受け取ることも可能です。 これは互換性を損なう変更が含まれる場合があるため推奨していません。_
+注: **非推奨** `@volatile` を使って最新バージョンの Orb を受け取ることも可能です。 これは互換性を損なう変更が含まれる場合があるため推奨していません。
```
Comment (rsylvian) [inline jekyll/_cci2_ja/orbs-faq.md:64]: Thanks @yuki-sakamoto-san for this fix! :)

File: jekyll/_cci2/api-developers-guide.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/api-developers-guide.md:12]: Ah I see!

File: jekyll/_cci2/projects.md (hunk 2 above)
Comment (rosieyohannan) [reply jekyll/_cci2/projects.md:14]: I was going to suggest we just remove the link altogether here but this page is actually about to be archived! (after we get this PR merged though)

File: jekyll/_cci2/projects.md (hunk 3 above)
Comment (rosieyohannan) [reply jekyll/_cci2/projects.md:29]: same ^

File: jekyll/_cci2/security-recommendations.md (hunk 4 above)
Comment (rsylvian) [reply jekyll/_cci2/security-recommendations.md:110]: sounds good!

File: jekyll/_includes/edit-on-github.html [hunk 10]
```diff
@@ -1,20 +0,0 @@
-<br />
```
[the remark this answers is not in the document]
Comment (rsylvian) [reply jekyll/_includes/edit-on-github.html:1]: It is still live but it is a `docs-platform` component: https://github.com/circleci/docs-platform/blob/main/src/components/app/Content/PageFooter.tsx

**Review by kbatuigas — APPROVED** [review]
🎉

File: jekyll/_cci2_ja/orbs-faq.md (hunk 9 above)
Comment (rsylvian) [reply jekyll/_cci2_ja/orbs-faq.md:64]: Screen Shot 2023-03-16 at 12 58 27 PM

**Pull request merged**